boards/nxp/mimxrt1170_evk/mimxrt1170_evk_mimxrt1176_cm7_B.overlay

```diff
@@ -25,18 +26,14 @@
 	w25q512nw:w25q512nw@0 {
 		/* IS25WP128 flash chip not currently enabled */
 		compatible = "nxp,imx-flexspi-nor";
-		size = <DT_SIZE_M(512)>;
+		size = <DT_SIZE_M(64*8)>;
```

Comment:
Minor request- could you remove the comment `/* IS25WP128 flash chip not currently enabled */`? I know this is out of scope for this PR, but that comment isn't correct any more (and the file is already being updated here)

Reply:
> Minor request- could you remove the comment `/* IS25WP128 flash chip not currently enabled */`? I know this is out of scope for this PR, but that comment isn't correct any more (and the file is already being updated here)

Done. The comment has been removed.